src/backend/tasks_io/datafeeds/parsers/fms_api/tests/fms_api_event_list_parser_test.py

```diff
@@ -532,6 +532,84 @@ def test_parse_division_parent_2018(test_data_importer):
             assert event.divisions == []
 
 
+def test_parse_division_parent_2023(test_data_importer):
+    path = test_data_importer._get_path(__file__, "data/2023_event_list.json")
+    with open(path, "r") as f:
+        data = json.load(f)
+    events, districts = FMSAPIEventListParser(2023).parse(data)
+    assert len(events) == 178
```

Comment:
Should this be 186?

Reply:
That's what I thought as well, but... `178` passes...  👀 

The test data is pulled directly from here:
https://frc-api.firstinspires.org/v3.0/2023/events

Reply:
178 passes because the 8 divisions aren't getting parsed.